<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PixelesMejorados</class>
 <widget class="QDialog" name="PixelesMejorados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Azul:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rojo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Verde:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>160</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="BotonIniciar">
      <property name="text">
       <string>INICIAR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BotonDetener">
      <property name="text">
       <string>DETENER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BotonSalir">
      <property name="text">
       <string>REGRESAR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tama√±o del pixel:</string>
   </property>
  </widget>
  <widget class="QDial" name="dialRojo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>453</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QDial" name="dialVerde">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>523</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QDial" name="dialAzul">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>590</y>
     <width>50</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRojo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxVerde">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAzul">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotonSalir</sender>
   <signal>clicked()</signal>
   <receiver>PixelesMejorados</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
